src/vs/workbench/contrib/scm/browser/scm.contribution.ts

```diff
@@ -227,7 +227,7 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
 				localize('scm.providerCountBadge.auto', "Only show count badge for Source Control Provider when non-zero."),
 				localize('scm.providerCountBadge.visible', "Show Source Control Provider count badges.")
 			],
-			description: localize('scm.providerCountBadge', "Controls the count badges on Source Control Provider headers. These headers only appear when there is more than one provider."),
+			markdownDescription: localize('scm.providerCountBadge', "Controls the count badges on Source Control Provider headers. These headers appear in the \"Source Control\", and \"Source Control Sync\" views when there is more than one provider or when the `#scm.alwaysShowRepositories#` setting is enabled, as well as in the \"Source Control Repositories\" view."),
```

Comment:
You should move the `#scm.alwaysShowRepositories#` part into a variable so translators don't break the link or setting name.

Reply:
@lszomoru This is the change required, how can I commit it in?

```
markdownDescription: localize('scm.providerCountBadge', "Controls the count badges on Source Control Provider headers. These headers appear in the \"Source Control\", and \"Source Control Sync\" views when there is more than one provider or when the {0} setting is enabled, as well as in the \"Source Control Repositories\" view.", `\`#scm.alwaysShowRepositories#\``),
```

Reply:
https://github.com/microsoft/vscode/pull/194967

Reply:
@lszomoru is my change ok?

Reply:
I can't test this until it is in insider or production